assignment/index.html

```diff
@@ -3,11 +3,16 @@
   <head>
     <meta charset="utf-8">
     <title>Unit 5, Part 4 - Music Collection</title>
-    <script src="scripts/3-music-collection.js" charset="utf-8"></script>
   </head>
   <body>
     <h1>Unit 5, Part 4 - Music Collection</h1>
     <p>See console for output.</p>
     <!-- Create an unordered list with any questions you have -->
+    <ul>
+      <li>This is random, but if I needed a quick refresher on semicolons, where could I go?</li>
```

Comment:
That's a great question. I don't know of a particularly good resource. It's something people mostly learn through practice I guess. When you start working with linters/formatters, they will automatically handle that for you to a certain extent too.